includes/base_controls/QControlProxy.class.php

```diff
@@ -62,7 +60,7 @@ public function RenderAsHref($strActionParameter = null, $blnDisplayOutput = tru
 			else
 				$this->strTargetControlId = $this->objForm->GenerateControlId();
 			
-			$this->strActionParameter = $strActionParameter;
+			$this->mixActionParameter = $strActionParameter;
```

Comment:
Its defined in QControlBase. The local $strActionParameter was redundant and not necessary. I checked in some minor changes to fix some PhpStorm warnings, but I did not see the warning you were talking about saying that it was undefined.